torchao/dtypes/affine_quantized_tensor.py

```diff
@@ -20,10 +20,32 @@
     _ATEN_OP_OR_TORCH_FN_TABLE,
     _register_layout_cls,
     _get_layout_tensor_constructor,
+    LayoutType,
 )
+from typing import ClassVar
+from dataclasses import dataclass
 
 aten = torch.ops.aten
 
+@dataclass(frozen=True)
+class PlainLayoutType(LayoutType):
```

Comment:
comment or error that this shouldnt be instantiated directly

Reply:
this can be instantiated I think, are you talking about LayoutType?

Reply:
I see I guess I'm a bit thrown off because a data classes primary goal is to store data wheras this class stores nothing

Reply:
I would have instead done an enum like this

```
from enum import Enum
class Operations(Enum):
    ADD = (1,)
    SUBTRACT = (2,)
    MULTIPLY = (3,)
    DIVIDE = (4, 'precision') 
```

enums are also a class so you can override `__init__` and define a `func` that only applies on DIVIDE for example

Reply:
sorry I don't follow the `DIVIDE` part, can you elaborate a bit? is this talking about how to support `TensorCoreTiledLayoutType` that has a inner_k_tiles argument?